src/cli.js

```diff
@@ -60,11 +61,32 @@ function noIssue(result) {
   );
 }
 
-function prettify(caption, deps, oneline) {
+function prettify(
+  caption,
+  deps,
+  oneline,
+  rootDir = null,
+  printJsonLineInKey = null,
```

Comment:
I added two parameters here:
- `rootDir` to open the `package.json`file,
- `printJsonLineInKey` is to find the extra dependency in either "dependencies" or "devDependencies"